mcs/class/corlib/coreclr/ConfiguredValueTaskAwaitable.cs

```diff
@@ -0,0 +1,88 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
We import corefx as a submodule.  Could we get this filre from there?

Reply:
Its the coreclr version for the internal `[StackTraceHidden]` annotations